sktime/alignment/base.py

```diff
@@ -336,4 +336,12 @@ def _get_distance_matrix(self):
         distmat: an (n x n) np.array of floats, where n is length of X passed to fit
             [i,j]-th entry is alignment distance between X[i] and X[j] passed to fit
         """
-        raise NotImplementedError
```

Comment:
You state that this is only sensible if two time series are aligned. Thus, I would suggest to raise an error if this method is called when aligning multiple time series.

Reply:
I see - perhaps it should simply return the notimplementederror in this case?

Reply:
ok, done